src/View/Home/index.html.twig

```diff
@@ -1,11 +1,10 @@
 {% extends 'layout.html.twig' %}
 
+{% block classBody %}indexBackground
+{% endblock %}
 {% block content %}
-	<h1>Hello Wilder !</h1>
-	<p>Find a complete CRUD example on
-		<a href="/concerts">/concerts</a>
-		page.</p>
-	<p>Find a complete CRUD example on
-		<a href="/artists">/artists</a>
-		page.</p>
+	<section class="indexSection">
+		<p class="logotxt">goldfish</p>
+		<img src="/assets/images/goldfish.png" class="logoimg" alt="...">
```

Comment:
If you have an alt on your img, you HAVE TO put some text in it

Reply:
fix alt image